trax/data/tf_inputs.py

```diff
@@ -1107,6 +1112,42 @@ def bert_double_sentence_preprocess(batch):
     yield value_vector, segment_embs, segment_embs, label, np.int64(1)
 
 
+def mask_random_tokens(batch, vocab_size=30522, masking_prob=0.15, merge_with_loss=True):
```

Comment:
Can you extend the comment and link the mask_random_tokens method in the BERT implementation you have in mind?